packages/react-composites/src/composites/MeetingComposite/NotificationIcon.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Icon, Stack } from '@fluentui/react';
+import { NotificationIconStyles, NotificationTextStyles } from './styles/MeetingCompositeStyles';
+
+/**
+ * @private
+ */
+export type NotificationIconProps = {
+  numberOfMessages: number;
+};
+/**
+ * @private
+ */
+export const NotificationIcon = (props: NotificationIconProps): JSX.Element => {
+  const { numberOfMessages } = props;
+  return (
+    <Stack horizontalAlign="center" verticalAlign="center" style={{ position: 'absolute', top: 0, right: '0.78rem' }}>
+      <Icon styles={NotificationIconStyles} iconName="NotificationCircle" />
+      <Stack styles={NotificationTextStyles}>
+        {numberOfMessages > 0 && numberOfMessages < 9 ? <>{numberOfMessages}</> : <></>}
```

Comment:
nit: I would simplify this and deal with not-valid values seperately, so here have:
```
{ numberOfMessages < 9 ? <>{numberOfMessages}</> : ? <>9+</>
```

And then near the top of this function have the explicit case:
```
if (numberOfMessages < 1) return <></>
```

It helps make the render portion easier to read, and helps easily understand what invalid consitions will lead to: `<></>`

Reply:
One extra pro of this is typescript will soon support narrowing of types! Meaning instead of 
```
  numberOfMessages: number;
```
we will be able to do:
```
  numberOfMessages: numberGreaterThanZero;
```

Reply:
Another suggestion would be to abstract all this logic in a separate function and call that function here instead.
This logic might grow in the future and will make it easier to scale.

Reply:
okay will look at this!